pstress/pstress_log_to_sql_converter.sh

```diff
@@ -46,11 +52,11 @@ echo "Converted SQL file can be found here: $outputFileName"
 # NOTE:                                                                                                         #
 # 1. Make sure to start a fresh MySQL instance before running this script. Executing the SQL file on an already #
 #    running server may have existing database objects (eg. general tablespaces, tables, etc) causing failures. #
-# 2. Comment SQL execution section(below) in case pstress logs are generated in multi-threaded mode. Reason     #
-#    being that, some database objects might be created in another thread causing SQLs to fail.                 #
 #################################################################################################################
-# Executing SQLs against a running server (Optional)
-BASEDIR=$HOME/mysql-8.0/bld_8.0.28/install
-SOCKET=/tmp/mysql_22000.sock
-$BASEDIR/bin/mysql -uroot -S$SOCKET -e "source $outputFileName"
+# To Execute SQLs against a running server set the BASEDIR and SOCKET path (Optional).
+BASEDIR=
+SOCKET=
+if [[ $BASEDIR != "" && $SOCKET != "" ]]; then
```

Comment:
It would caught trouble if some want to execute from build directory. Can you check how scripts work which can execute mysql from build or install directory.

Reply:
For running pstress also, we use the same config variable BASEDIR inside pstress-run.conf. This script is also going to be at the same place. Hence, we should ask the user to update the BASEDIR path in case he wants to execute the generated SQL.

A user may want to execute the script from anywhere not only from build directory. It is not right for the script to find the build path on its own. On a machine there can be multiple build locations and multiple type of builds (out-of-source builds, built from packages etc)

Reply:
it a bash script, so base directory is not required, since it doesn't need any .so which is required for c++ binaries like pstress.
The end  purpose of this script is to execute the sql generated by pstress in multiple thread. so just pass the mysql path is sufficient which make this robust to  run through build or install directory.

Reply:
I think the name BASEDIR is confusing. Maybe I should rename it to BUILD_DIR
We are using $BASEDIR/bin/mysql to find mysql binaries.

Reply:
if you are running from build directory it is not in bin directory. it is in ./runtime_output_directory/mysql.